analytics_data_api/v0/views/learners.py

```diff
@@ -85,55 +94,66 @@ class LearnerListView(CourseViewMixin, generics.ListAPIView):
+              work in this course (for example, "highly_engaged" or
+              "struggling").
             * engagements: Summary of engagement events for a time span.
-                * videos_viewed: Number of times a video was played.
-                * problems_completed: Unique number of problems completed.
-                * problems_attempted: Unique number of problems attempted.
-                * discussions_contributed: Number of discussions (e.g. forum posts).
+                * videos_viewed: Number of times any course video was played.
+                * problems_completed: Number of unique problems the learner
+                  answered correctly.
+                * problems_attempted: Number of unique problems attempted.
+                  This is a count of the individual problems the learner
+                  tried. Each problem in a course can increment this count by
+                  a maximum of 1.
+                * discussions_contributed: Number of posts, responses, or
+                  comments the learner contributed to course discussions.
 
     **Parameters**
 
-        You can filter the list of learners by course ID and other parameters
-        such as enrollment mode and text search.  You can also control the
-        page size and page number of the response, as well as sort the learners
-        in the response.
-
-        course_id -- The course within which user data is requested.
-        page -- The page of results which should be returned.
-        page_size -- The maximum number of results which should be returned per page.
-        text_search -- A string to search over the name, username, and email of learners.
-        segments -- A comma-separated string of segments to which
-            learners should belong.  Semgents are "OR"-ed together.
-            Cannot use in combination with `ignore_segments`
-            argument.
-        ignore_segments -- A comma-separated string of segments to
-            which learners should NOT belong.  Semgents are "OR"-ed
-            together.  Cannot use in combination with `segments`
-            argument.
+        You can filter the list of learners by course ID and by other
+        parameters, including enrollment mode and text search. You can also
+        control the page size and page number of the response, as well as sort
+        the learners in the response.
+
+        course_id -- The course identifier for which user data is requested.
+            For example, edX/DemoX/Demo_Course.
+        page -- The page of results that should be returned.
+        page_size -- The maximum number of results to return per page.
+        text_search -- An alphanumeric string that is used to search name,
+            username, and email address values to find learners.
+        segments -- A comma-separated list of segment names that is used
+            to select learners. Only learners who are categorized in one of
```

Comment:
Thanks! A nitpick: learners who are categorized in *at least one* of the segments are returned. Another way of saying this is that returned learners may be in more than one of the segments as opposed to exactly one of the segments. I understand if you don't feel the need to explicitly write this, but thought I'd point it out just in case.